moveit_ros_control_interface/moveit_core_plugins.xml

```diff
@@ -0,0 +1,10 @@
+<class_libraries>
```

Comment:
Why did you call this file ``moveit_core_plugins.xml``? Seems like it should be ``moveit_ros_control_interface_plugin_description.xml`` to follow standards